scripts/branches

```diff
@@ -4,20 +4,32 @@ import argparse
 import os.path
 import subprocess
 import sys
-
+import json
 from lib import foreach_repo, github
 
+out = {}
 parser = argparse.ArgumentParser(description="Show branches of Pop!_OS repositories")
 parser.add_argument("repos", nargs="*", default=[])
-args = parser.parse_args(sys.argv[1:])
+parser.add_argument("--output",help="Set output format (Currently only json)")
+args = parser.parse_args()
 
 def callback(repo):
-    print("\x1B[1m" + repo["name"] + "\x1B[0m")
-
-    for branch in github(repo["branches_url"].replace("{/branch}", "")):
-        if branch["name"] == repo["default_branch"]:
-            print(" * " + branch["name"])
-        else:
-            print("   " + branch["name"])
+	if args.output == None:
+		print("\x1B[1m" + repo["name"] + "\x1B[0m")
+	else:
+		out[repo["name"]] = []
+	for branch in github(repo["branches_url"].replace("{/branch}", "")):
+		if branch["name"] == repo["default_branch"]:
+			if args.output == None:
+				print(" * " + branch["name"])
+			else:
+				out[repo["name"]].append(branch["name"])
+		else:
+			if args.output == None:
```

Comment:
This else clause is almost identical to the previous. This should be possible without nesting nearly identical if statements.

Reply:
Probably something like 
```
spacer = " * " if branch["name"] == repo["default_branch"] else "   "
if args.output == None:
    print(spacer + branch["name"])
else:
    out[repo["name"]].append(branch["name"])
```

Reply:
Already changed, :+1:

Reply:
Setting the `spacer` or whatever you want to call it is still shorter, but I'm not going to nit pick too much. What you had before wasn't a styleistic choice, what was there before was just bad not DRY code.

Reply:
That's fair